atomic_reactor/constants.py

```diff
@@ -73,3 +73,4 @@
 PLUGIN_ADD_FILESYSTEM_KEY = 'add_filesystem'
 PLUGIN_FETCH_WORKER_METADATA_KEY = 'fetch_worker_metadata'
 PLUGIN_GROUP_MANIFESTS_KEY = 'group_manifests'
+PLUGIN_BUILD_ORCHESTRATE_KEY = 'orchestrate_build'
```

Comment:
orchestrate build plugin should also use this.

Reply:
Fixed